src/custom/components/Version/index.tsx

```diff
@@ -36,19 +32,25 @@ const VERSIONS: Record<
   'Vault Relayer': {
     version: 'v' + CONTRACTS_VERSION,
     href(chainId: ChainId) {
-      return {
-        etherscan: _getContractsUrls(chainId, GP_VAULT_RELAYER),
-        github: `https://github.com/cowprotocol/contracts/blob/v${CONTRACTS_VERSION}/src/contracts/GPv2VaultRelayer.sol`,
-      }
+      // return Etherscan by default
+      return _getContractsUrls(chainId, GP_VAULT_RELAYER)
+
+      // return {
+      //   etherscan: _getContractsUrls(chainId, GP_VAULT_RELAYER),
+      //   github: `https://github.com/cowprotocol/contracts/blob/v${CONTRACTS_VERSION}/src/contracts/GPv2VaultRelayer.sol`,
+      // }
```

Comment:
Is this only temporary or do you intent to remove the github link from the page?

Reply:
Yes my suggestion here is to pick one external link/url. IMO more links should be part of a FAQ or a dedicated page. But overall I think linking directly to the smart contract is best practice. Open to other suggestions here.